client/rest-high-level/src/main/java/org/elasticsearch/client/RequestConverters.java

```diff
@@ -1007,6 +1008,18 @@ static Request getTemplates(GetIndexTemplatesRequest getIndexTemplatesRequest) t
         return request;
     }
 
+    static Request analyze(AnalyzeRequest request) throws IOException {
+        EndpointBuilder builder = new EndpointBuilder();
+        String index = request.index();
+        if (index != null) {
+            builder.addPathPart(index);
+        }
```

Comment:
question: I see in the REST spec that we have also prefer_local and format. I don't see them supported in the corresponding REST action though. Can you double check? Maybe those params should be removed from the SPEC?

Reply:
sounds good thanks!